README.md

```diff
@@ -93,15 +93,25 @@ Once the page is loaded, the browser sends further async requests as needed.
 20. ### What is a Higher Order Component ? Give example and explain the use cases where it should be used.
      Answer To be added
 21. ### How to deep clone a object in javascript ?
-     Answer To be added
+     Assuming that you only have variables and not have any functions in your object,just use:
```

Comment:
@darshant31 Request you to add a more comprehensive answer that would be required in a real interview. Of course, you could start with this answer, however, the interviewer would expect person to provide a solution that works in all cases without these assumptions.

Reply:
sure

Reply:
done,please have a look at it